addon/validations/factory.js

```diff
@@ -98,14 +99,27 @@ function buildValidations(validations = {}, globalOptions = {}) {
   let Validations;
 
   return Ember.Mixin.create({
-    _validationsClass: computed(function () {
+    init() {
+      this._super(...arguments);
+
+      // Count number of mixins to bypass super check if there is more than 1
+      this.__validationsMixinCount__ = this.__validationsMixinCount__|| 0;
+      this.__validationsMixinCount__++;
```

Comment:
Small tweak here, we should store the resulting integer in a closure scoped variable:

```js
let Validations, validationMixinIndex;
// ...snip...

validationMixinIndex = this.__validationsMixinCount__++;
```

Then below, the guard for calling super in `__validationsClass__` would look like:

```js
if(shouldCallSuper(this, '__validationsClass__') || validationMixinIndex > 1) {
  inheritedClass = this._super();
}
```

This ensures that the first mixin with a `__validationsClass__` only calls `_super` if the superclass passes the `shouldCallSuper` check, and the second (and any more) mixins always call `_super`.